pyro_risks/platform_fwi/get_fwi_effis_score.py

```diff
@@ -0,0 +1,47 @@
+# Usual Imports
```

Comment:
[LOW] IMO these comments are not needed when importing libraries (maybe use `isort` to automatically sort your imports.

Reply:
done, thanks for the tip !